add-evens/starter.py

```diff
@@ -0,0 +1 @@
+def add_evens(numbers) -> list:
```

Comment:
You should specify the type for the numbers parameter.

Reply:
nah, you don't need to, and that's not the correct syntax for list typing 

I removed the type annotations

otherwise LGTM